scene/animation/animation_blend_tree.h

```diff
@@ -36,19 +36,30 @@
 class AnimationNodeAnimation : public AnimationRootNode {
 	GDCLASS(AnimationNodeAnimation, AnimationRootNode);
 
+public:
+	enum PlayMode {
+		PLAY_MODE_FORWARD,
+		PLAY_MODE_BACKWARD
+	};
+
+	enum LoopMode {
+		LOOP_NO_OVERRIDE,
+		LOOP_NONE,
+		LOOP_LINEAR,
+		LOOP_PINGPONG
+	};
```

Comment:
To keep compatibility with the Animation.LoopMode enum, `LOOP_NO_OVERRIDE` should be the last element (3)

We may want to consider adding `LOOP_NO_OVERRIDE` to Animation.LoopMode and then directly referencing Animation.LoopMode

Reply:
Alright, done.